bot/utils/decorators.py

```diff
@@ -2,9 +2,9 @@
 import functools
 import logging
 import random
-import typing as t
 from asyncio import Lock
 from functools import wraps
+from typing import Callable, Container, Union
```

Comment:
`Container` should be imported from `collections.abc` since it's deprecated in `typing`.